book3/13-web.mkd

```diff
@@ -253,152 +253,153 @@ crea un conjunto de *servicios* disponibles para que los usen otras aplicaciones
+geográfica, como “Ann Arbor, MI” a su API de geocodificación y conseguir que
+Google nos devuelva su mejor suposición sobre donde podría estar nuestra cadena
+de búsqueda en un mapa, además de los puntos de referencia en los alrededores.
 
-The geocoding service is free but rate limited so you cannot make
-unlimited use of the API in a commercial application. But if you have
-some survey data where an end user has entered a location in a
-free-format input box, you can use this API to clean up your data quite
-nicely.
+El servicio de geocodificación es gratuito, pero limitado, de modo que no se puede
+hacer un uso intensivo de esta API en una aplicación comercial. Pero si tienes ciertos
+datos estadísticos en los cuales un usuario final ha introducido una localización
+en formato libre en un cuadro de texto, puedes utilizar esta API para limpiar esos
+datos de forma bastante efectiva.
 
-*When you are using a free API like Google's geocoding API, you
-need to be respectful in your use of these resources. If too many people
-abuse the service, Google might drop or significantly curtail its free
-service.*
+*Cuando se usa una API libre, como la API de geocodificación de Google, se debe
+ser respetuoso con el uso de los recursos. Si hay demasiada gente que abusa del
+servicio, Google puede interrumpir o restringir significativamente su uso gratuito.*
 
-\index{rate limiting}
+\index{limitar uso}
 
-You can read the online documentation for this service, but it is quite
-simple and you can even test it using a browser by typing the following
-URL into your browser:
+Puedes leer la documentación online de este servicio, pero es bastante sencillo y
+puedes incluso probarlo desde un navegador, simplemente tecleando la siguiente
+URL en él:
 
 [http://maps.googleapis.com/maps/api/geocode/json?address=Ann+Arbor%2C+MI](http://maps.googleapis.com/maps/api/geocode/json?address=Ann+Arbor%2C+MI)
 
-Make sure to unwrap the URL and remove any spaces from the URL before
-pasting it into your browser.
+Asegúrate de limpiar la URL y eliminar cualquier espacio de ella antes de pegarla
+en tu navegador.
 
-The following is a simple application to prompt the user for a search
-string, call the Google geocoding API, and extract information from the
-returned JSON.
+La siguiente es una aplicación sencilla que pide al usuario una cadena de búsqueda,
+llama a la API de geocodificación de Google y extrae información del JSON que
+nos devuelve.
 
 \VerbatimInput{../code3/geojson.py} 
 
-The program takes the search string and constructs a URL with the search
-string as a properly encoded parameter and then uses
-`urllib` to retrieve the text from the Google geocoding
-API. Unlike a fixed web page, the data we get depends on the parameters
-we send and the geographical data stored in Google's servers.
+El programa toma la cadena de búsqueda y construye una URL codificándola como
+parámetro dentro de ella, utilizando luego `urllib ` para recuperar el texto de la
```

Comment:
Replace this single quotes to the English keyboard ones, 'urllib ' instead of `urllib `.